_docs_integrate/data-model-overview.md

```diff
@@ -336,8 +336,12 @@ The [creation of a RelationshipAttribute]({% link _docs_integrate/create-attribu
 - We refer to the LocalAttribute of the `owner` as an **own shared RelationshipAttribute**.
 - The peer’s LocalAttribute is referred to as a **peer shared RelationshipAttribute**.
 
-If the underlying RelationshipAttribute of an own shared RelationshipAttribute is shared with a peer who is not involved in the Relationship in which the RelationshipAttribute exists, this results in the creation of two LocalAttributes, an **own shared ThirdPartyRelationshipAttribute** on the one hand and a **peer shared ThirdPartyRelationshipAttribute** on the other.
-The two LocalAttributes that are created when sharing an underlying RelationshipAttribute of a peer shared RelationshipAttribute with a peer who is not involved in the Relationship in which the RelationshipAttribute exists are both referred to as a **third party owned RelationshipAttribute**.
+If a RelationshipAttribute is forwarded to a peer who is not involved in the Relationship in which the RelationshipAttribute exists, this will result in the creation of a pair of ThirdPartyRelationshipAttributes.
```

Comment:
Mostly, we are using "which" and not "who" when referring to peers. (Same below with "whom".)